src/index.html

```diff
@@ -0,0 +1,7 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Page Title</title>
+  </head>
+  <body>Hello, world!</body>
+</html>
```

Comment:
why didn't you use the default emmet layout? (by typing `!`)

Reply:
yep, you are right that would be easier